include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -272,11 +274,21 @@ def XTenNN_LoadExternalConstOp: XTenNN_Op<"load_external_const", [
   }];
   let arguments = (ins
     StrAttr:$key,
-    StrAttr:$file
+    StrAttr:$file,
+    OptionalAttr<StrAttr>:$method
```

Comment:
I am wondering if `method` should not be a default attr, defaulting to h5. (But this would require overwriting the printer if we want to keep the IR backwards compatible)